index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
```

Comment:
Yes, but still breaking change, previously passing opts.fields=true would have returned the raw doc: https://github.com/digidem/osm-p2p-db/blob/3.10.0/index.js#L217

With this change it would not. I don't know if anything is relying on this, at least nothing in osm-p2p-server code.